lib/apiservers/engine/backends/portlayer/container_proxy.go

```diff
@@ -265,14 +273,58 @@ func (c *ContainerProxy) AddVolumesToContainer(handle string, config types.Conta
 	return handle, nil
 }
 
-// CommitContainerHandle() commits any changes to container handle.
+// AddLoggingToContainer adds logging capability to a container, referenced by handle.
+// If an error is return, the returned handle should not be used.
+//
+// returns:
+//	modified handle
+func (c *ContainerProxy) AddLoggingToContainer(handle string, config types.ContainerCreateConfig) (string, error) {
+	defer trace.End(trace.Begin(handle))
+
+	if c.client == nil {
+		return "", InternalServerError("ContainerProxy.AddLoggingToContainer failed to create a portlayer client")
```

Comment:
Pedantic - this isn't creating a portlayer client, but instead attempting to use a pre-created one.

For future work - we shouldn't even get to these calls if there's no port layer client created.